featurebyte/routes/target/api.py

```diff
@@ -74,25 +75,8 @@ async def get_target(request: Request, target_id: PydanticObjectId) -> TargetMod
     """
     Retrieve Target
     """
-    controller = request.state.app_container.target_controller
-    target: TargetModel = await controller.get(
-        document_id=target_id,
-    )
-    return target
-
-
-@router.patch("/{target_id}")
-async def update_target(
-    request: Request,
-    target_id: PydanticObjectId,
-    data: TargetUpdate,
-) -> TargetModel:
-    """
-    Update Target
-    """
-    controller = request.state.app_container.target_controller
-    target = await controller.target_service.update_document(target_id, data)
```

Comment:
This was wrong as `controller` doesn't have the `target_service` attribute. This was untested, and it shouldn't exist anyway since target name should not be updated.